recipe/meta.yaml

```diff
@@ -29,14 +29,14 @@ requirements:
     - click-option-group
     - rdkit
     - openff-utilities
-    - openff-toolkit-base >=0.10.0,<0.11.0a0
+    - openff-toolkit-base >=0.11
     - openff-forcefields
     - openff-qcsubmit
-    - forcebalance >=1.9.2
-    - openff-fragmenter-base >=0.1.2
-    - openmm >=7.6.0
+    - forcebalance
+    - openff-fragmenter-base
+    - openmm
     - qcportal >=0.15.6
-    - qcengine >=0.23.0
+    - qcengine >=0.25
```

Comment:
The toolkit constraint is not pushed to their packaging, but this is the version that should be compatible

https://github.com/MolSSI/QCEngine/blob/master/docs/source/changelog.rst#v0250--2022-11-11